client/app-context/UserSelectionContext/types.ts

```diff
@@ -12,7 +12,7 @@ export type AddressOption = {
 	label: string;
 	coordinates: Coordinates;
 };
-export type TResult = { facilities: { id: number; distance: number }[] };
+export type TResult = { facilities: { id: number; distance: number; latlng: Array<number> }[] };
```

Comment:
i'm imagining that `latlng` will be `[lat,lng]`, and if so, should be typed as such

Reply:
Will be using LatLngExpression from leaflet